.gitignore

```diff
@@ -0,0 +1,8 @@
+/target/
+/.classpath
+/.project
+/.settings
+/nbactions.xml
+/nb-configuration.xml
+/bin/
+webapp/WEB-INF/conf/db.properties
```

Comment:
This is not a standard .gitignore file, 
(it  is not necessary to provide a .gitignore file)